samples/v1.3/Scenarios/ActivityUpdate.json

```diff
@@ -0,0 +1,118 @@
+{
```

Comment:
We also need to update the templated version of `Scenarios` cards

Reply:
Do we have some data to update the templated versions or do we have to create new data cards?

Reply:
I think we already have some data you can use (see [here](https://github.com/microsoft/AdaptiveCards/blob/main/samples/Templates/Scenarios/ActivityUpdate.data.json))

Reply:
Fixed